assets/models/system/Llama-2-13b-chat/spec.yaml

```diff
@@ -43,5 +43,6 @@ tags:
   inference_supported_envs:
     - vllm
     - ds_mii
+  benchmark: "quality"
```

Comment:
what other values are supported?

Reply:
currently quality and embeddings.  Please see the spec here:
https://msdata.visualstudio.com/Vienna/_git/vienna-wiki?path=%2FExperiences%2FProductions%2FDesign-Docs%2Ffoundation-models%2Fft-eval-contracts.md&version=GBmaster&_a=preview&anchor=required-tags

```
(OPTIONAL) benchmarks: list of supported benchmarks which will render a benchmarks icon in model catalog for a specific model.

For benchmarks, we can start with the following two groups of benchmarks (following the existing two tabs): quality, embeddings. The presence of the benchmarks key will correspond to showing an icon in the model catalog.
```